doc/opencv.bib

```diff
@@ -0,0 +1,907 @@
+@comment{Bib-it,
```

Comment:
You should use module file instead: https://github.com/opencv/opencv_contrib/blob/master/modules/ximgproc/doc/ximgproc.bib